pkg/dynatrace/dynatrace.go

```diff
@@ -195,28 +163,50 @@ func (dynatraceApiClient *DynatraceApiClient) GetDynatraceEquivalentClusterRegio
 
 	/*return location id from response body in which dynatrace location is public && CloudPlatform is AWS/AMAZON_EC2
 	e.g. returns exampleLocationId for N. Virginia location in dynatrace in which CloudPlatform is AWS and location is public
-	e.g. response body
+	e.g. PublicAndPrivate response body
 	{
 		"name": "N. Virginia",
 		"entityId": "exampleLocationId",
 		"type": "PUBLIC",
 		"cloudPlatform": "AMAZON_EC2",
+		"status": "ENABLED"
 	}*/
+
+	/*
+		e.g. Private Response body
+		{
+			"name": "backplanei03xyz",
+			"entityId": "privateLocationId",
+			"type": "PRIVATE",
+			"status": "ENABLED"
+		},
+	*/
+	// Decode the response body
 	var locationResponse DynatraceLocation
 	err = json.NewDecoder(resp.Body).Decode(&locationResponse)
 	if err != nil {
 		return "", err
 	}
-	for _, loc := range locationResponse.Locations {
-		if loc.Name == dynatraceLocationName && loc.Type == "PUBLIC" && loc.CloudPlatform == "AMAZON_EC2" {
-			return loc.EntityId, nil
+
+	if locationType == "PublicAndPrivate" {
+		for _, loc := range locationResponse.Locations {
+			if loc.Name == locationName && loc.Type == "PUBLIC" && loc.CloudPlatform == "AMAZON_EC2" && loc.Status == "ENABLED" {
+				return loc.EntityId, nil
+			}
+		}
+	}
+	if locationType == "Private" {
```

Comment:
Same here - can we use the [hypershift-defined values](https://github.com/openshift/hypershift/blob/main/api/hypershift/v1alpha1/hostedcluster_types.go#L1042-L1056) instead of hard-coded strings?

Reply:
Thanks! Now we're using `hypershiftv1beta1.PublicAndPrivate` and `hypershiftv1beta1.Private` instead of "PublicAndPrivate" and `"Private"`.